packages/app-form-builder/src/admin/plugins/editor/formFields/components/OptionsList.tsx

```diff
@@ -163,7 +163,7 @@ const OptionsList: React.FC<OptionsListProps> = ({ form, multiple, otherOption }
                         setOptionsValue(newValue);
                         clearEditOption();
                     },
-                    [optionsValue, setOptionsValue]
+                    [optionsValue, editOption, setOptionsValue]
```

Comment:
You know what... Let's loose the whole `useCallback` wrapper. I'm surprised this even works: a `useCallback` called in the `Bind` component's children renderer prop. Sheesh 😰